### DISCUSSION TITLE
why assign the  nearest vehile？

### DISCUSSION THREAD

**Discussion by lmmir** [openTCS/opentcs#178] [Q&A]
![image](https://github.com/user-attachments/assets/c32db429-bdad-4743-8af2-5d6640921f09)
to goods-out 02
why assign to vehicle-01 ,not vehcile-04?

**Comment by swltr** [comment]
@lmmir I assume you are using the default dispatcher. Which vehicle is assigned which transport order depends on how the default dispatcher is configured - see the [user's guide](https://www.opentcs.org/docs/6/user/opentcs-users-guide.html#_default_dispatcher).
